tools/lint-commits

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This file is copied from the original tools/commit-message-lint at zulip/zulip.
```

Comment:
It is copied, but also edited (line 14).

Reply:
Yeah, am looking for more edits, the git lint doesn't seem to work for GH actions. Shall we remove it from static tests?